docs/getting-started/installation.rst

```diff
@@ -23,11 +17,6 @@ Prerequisites
 
 - (Optional) OpenGL (to render gym environments)
 - (Optional) FFmpeg (to encode videos of renders)
-- (Optional) MuJoCo (follow instructions to install `mujoco\_py v1.5 here`_)
```

Comment:
I don't understand why MuJoCo has been removed here. Is MuJoCo incompatible with Gymnasium, and that's why it was removed?

I see MuJoCo still in some other dependencies, which makes me confused.

Reply:
MuJoCo is still a dependency. We used to depend on the `mujoco-py` package, which is requires manual installation steps. Now we depend on the `mujoco` package, which does not need manual installation steps. Therefore it has been removed from the installation instructions.

Reply:
Makes sense, thanks!